Eevee.Sleep.Bot/Extensions/DictionaryExtension.cs

```diff
@@ -0,0 +1,9 @@
+namespace Eevee.Sleep.Bot.Extensions;
+
+public static class DictionaryExtensions {
+    public static string GetString<K,V>(this IDictionary<K,V?> dict) {
```

Comment:
Name it (also change the behavior) to print something JSON-like or exactly JSON-looking with formatting instead.